src/_posts/databases/2000-01-01-maintenance-windows.md

```diff
@@ -0,0 +1,236 @@
+   months is displayed in the **Maintenance operations list** block
+
+### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo command line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, run the following command to list maintenance
+   operations:
+   ```bash
+   scalingo --app my-app --addon <database_type> database-maintenance-list
+   ```
+   With:
+   - `database_type`: must be either `postgresql`, `mysql`, `redis`, `mongodb`,
+     `elasticsearch` or `influxdb`, depending on the database addon
+
+   The output should look like this:
+   ```text
+   +--------------------------+-------------------+---------------------+---------------------+--------+
+   |            ID            |       TYPE        |     STARTED AT      |      ENDED AT       | STATUS |
+   +--------------------------+-------------------+---------------------+---------------------+--------+
+   | 65143fd19307d522665facc0 | db-node-migration | 2023/09/29 13:00:00 | 2023/09/29 13:00:00 | done   |
+   +--------------------------+-------------------+---------------------+---------------------+--------+
+   ```
+3. From the command line, run the following command to get more details about a
+   specific maintenance operation:
+   ```bash
+   scalingo --app my-app --addon <database_type> database-maintenance-info <maintenance_uuid>
+   ```
+   With:
+   - `database_type`: must be either `postgresql`, `mysql`, `redis`, `mongodb`,
+     `elasticsearch` or `influxdb`, depending on the database addon
+   - `maintenance_uuid`: id of the maintenance operation
+
+   The output should look like this:
+   ```
+   +------------+----------------------------------------------+
+   | ID         | 656ddbd39307d5ec79ff748b                     |
+   | Type       | db-node-migration                            |
+   | Started At | Next Maintenance Window: 2023/12/08 12:00:00 |
+   | Ended At   |                                              |
+   | Status     | scheduled                                    |
+   +------------+----------------------------------------------+
+   ```
+
+## Sharing the Information With Your Team
+
+By default, owner and collaborators receive email notifications one day before
+a scheduled maintenance execution. This notification system operates via the
+[App notifications](/platform/app/notification) feature and the "default
+notifier" which is configured for each app. Please check if it is still active
+or configure another notifier for this purpose.
+It can easily be configured to suit your preferences. For instance, if you
+prefer webhook notifications over email, you can easily configure this setting
+(or create a new dedicated notifier). Additionally, you have the option to
+filter the list of recipients or provide a service address for individuals who
+need to be notified but do not have a Scalingo account.
+
+Three new events have been added:
```

Comment:
issue: I don't think there is a concept of something "new" in such documentation page. I mean, in a 5 years, will it still make sense that they are "new"?
 ```suggestion
There are three events about database maintenance:
```